src/Compilers/CSharp/Portable/Symbols/AbstractTypeMap.cs

```diff
@@ -125,7 +125,7 @@ internal TypeSymbolWithAnnotations SubstituteType(TypeSymbol previous)
                     break;
             }
 
-            return TypeSymbolWithAnnotations.Create(result);
+            return TypeSymbolWithAnnotations.Create(result, ImmutableArray<CustomModifier>.Empty, isNullableIfReferenceType: null);
```

Comment:
>ImmutableArray<CustomModifier>.Empty [](start = 60, length = 36)

Why pass this explicitly, it looks like there is an overload that doesn't require modifiers.